NachoSpigot-API/pom.xml

```diff
@@ -10,29 +10,19 @@
     </parent>
 
     <artifactId>api</artifactId>
-    <version>1.8.8-R0.2-SNAPSHOT</version>
+    <version>1.8.8-R0.3-SNAPSHOT</version>
```

Comment:
Why the change?

Reply:
That's actually the latest 1.8.8 release, I'm pretty sure putting it as `R0.2` was an accident

Reply:
Spigot and Paper both have RO.1 but TacoSpigot [changed it](https://github.com/TacoSpigot/TacoSpigot/commit/816cde55dfc03d813ca55e0d66a49eb1fae11f21#diff-3e189d8b27d4e715e93f4f3e6f2875558d4f3f01d5a4526c9b7a1eed3dff3d54) to RO.2. I have no idea how the versioning works but I think it should be RO.1

Reply:
The sub-release of a version

Reply:
Then it should be RO.1